### PR TITLE
Add what up

### PR DESCRIPTION
**Pull request by joshling1919**
Add 'what up' div to board component in order to greet the user.

### CODE REVIEW

File: react-minesweeper/components/board.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Board extends React.Component{
     const that = this;
     return(
       <div id="board">
+        <div>What up</div>
```
Comment (amaciver) [inline react-minesweeper/components/board.jsx]: Sick addition!

**Review by amaciver — CHANGES_REQUESTED** [review]
AAAActually. no. add an exclamation!!!!

**Comment by joshling1919** [comment]
@amaciver added exclamation points! please review.

**Review by amaciver — APPROVED** [review]
THnku

**Pull request merged**